### PR TITLE
Modified service.py and tcpdriver.py for Keystone and Horizon.

### PR DESCRIPTION
**Pull request by ChristopherMSpencer**
The change being made is for changing the service names we are
targetting during service restarts as well as killing a service.
Also changed method for how we find the desired nodes.

 Please enter the commit message for your changes. Lines starting

### CODE REVIEW

File: os_faults/drivers/tcpcloud.py [hunk 1]
```diff
@@ -203,13 +216,13 @@ def verify(self):
...
         if not self.cached_cloud_hosts:
-            cmd = "salt -E '(ctl*|cmp*)' network.interfaces --out=yaml"
+            cmd = "salt -E '(infra*)' network.interfaces --out=yaml"
             result = self.execute_on_master_node({'command': cmd})
             stdout = result[0].payload['stdout']
             for fqdn, net_data in yaml.load(stdout).items():
                 host = node_collection.Host(
-                    ip=net_data['eth0']['inet'][0]['address'],
-                    mac=net_data['eth0']['hwaddr'],
+                    ip=net_data['eth1']['inet'][0]['address'],
```
Comment (ldcastell) [inline os_faults/drivers/tcpcloud.py:224]: should this part of the code be enclosed in a try block? what if the 'eth1' key does not exists? this is something carried from the code base but just wondering :)

**Review by ldcastell — APPROVED** [review]
looks good to me, just left a comment out there

File: os_faults/drivers/tcpcloud.py (hunk 1 above)
Comment (sumantmurke) [reply os_faults/drivers/tcpcloud.py:224]: I am not sure if tcpcloud requires eth1. As we are using OSA deployed cloud we might want eth1 but not sure if this change would be relevant to tcpcloud config.  I would suggest not make any changes to tcpcloud driver.

**Comment by mwturvey** [comment]
-2  Not going to merge this to master.  Will merge to osic-master instead

**Pull request closed without merging**